src/components/pagination/__tests__/Pagination.Test.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import { render, screen } from '@testing-library/react';
+
+import Pagination from '../index';
+
+const mockOnChange = jest.fn();
+const renderWrapper = (count = 5, page = 1) => {
+  return render(
+    <Pagination count={count} page={page} onChange={mockOnChange} />
+  );
+};
+
+describe('<Pagination />', () => {
+  it('should render with next/prev buttons', async () => {
+    renderWrapper();
+
+    const prev = screen.getByTestId('prev-button');
```

Comment:
could you try to use `getByRole` selector and use an `aria-label` for next/prev ? This improves accessibility a lot for buttons with only an icon